includes/user-functions.php

```diff
@@ -38,7 +37,14 @@ function edd_get_users_purchases( $user = 0, $number = 20, $pagination = false,
 		return false;
 	}
 
-	$status = $status === 'complete' ? 'publish' : $status;
+	if ( ! is_array( $status ) && is_string( $status ) ) {
```

Comment:
Yeah the `! is_array` isn't needed, should we support a comma separate list of statuses here too? So something like:
```
if ( is_string( $status ) ) {
    if ( strpos( ',', $status ) ) {
        $status = explode( ',', $status );
    } else {
        $status = $status === 'complete' ? 'publish' : $status;
        $status = array( $status );
    }
}